prime-angular-frontend/src/app/modules/enrolment/pages/access-agreement-history-enrolment/access-agreement-history-enrolment.component.ts

```diff
@@ -36,6 +38,14 @@ export class AccessAgreementHistoryEnrolmentComponent extends BaseEnrolmentPage
   public ngOnInit() {
     const enrolleeId = this.enrolmentService.enrolment.id;
     const accessTermId = this.route.snapshot.params.id;
+    const enrolment = this.enrolmentService.enrolment;
+
+    if (enrolment.expiryDate) {
+      const expiryMoment = moment(enrolment.expiryDate);
+      this.expiryDate = expiryMoment.isAfter(moment.now())
+        ? expiryMoment.format('MMMM Do, YYYY') : null;
```

Comment:
The ticket doesn't seem to specify, but would we want to hide the day it expired even if it has expired?

Reply:
No I dont think so, good catch. Although on the pharmanet-enrolment-summary i will not be able to show it even if it expired because the 'You Have Been Approved' alert box only shows if toa is active.